pepper-apis/src/main/java/org/broadinstitute/ddp/model/dsm/KitReasonType.java

```diff
@@ -0,0 +1,14 @@
+package org.broadinstitute.ddp.model.dsm;
+
+public enum KitReasonType {
```

Comment:
The kit reasons are still not really final. Also waiting for input from @pegahtah.

Reply:
Update: kit reasons mostly finalized.